.github/ISSUE_TEMPLATE/02-bug.yml

```diff
@@ -0,0 +1,51 @@
+name: Bug Report
```

Comment:
There is some error in this file fix that: https://github.com/rt4914/KlinicCon-Backend/blob/5ec1b1fb286869e7a5fa43e1c45a23190b36fcb8/.github/ISSUE_TEMPLATE/02-bug.yml

Reply:
Because of this error the UI is not getting generated correctly.